plenum/test/view_change/test_view_change_with_different_ic.py

```diff
@@ -0,0 +1,106 @@
+import pytest as pytest
+
+from plenum.test.node_catchup.helper import ensure_all_nodes_have_same_data, waitNodeDataEquality
+from plenum.test.pool_transactions.helper import disconnect_node_and_ensure_disconnected
+from plenum.test.test_node import ensureElectionsDone, checkNodesConnected
+from plenum.test.view_change.helper import start_stopped_node
+from stp_core.loop.eventually import eventually
+
+
+@pytest.fixture(scope="module")
+def tconf(tconf):
+    old_interval = tconf.OUTDATED_INSTANCE_CHANGES_CHECK_INTERVAL
+
+    tconf.OUTDATED_INSTANCE_CHANGES_CHECK_INTERVAL = 5
+    yield tconf
+
+    tconf.OUTDATED_INSTANCE_CHANGES_CHECK_INTERVAL = old_interval
+
+
+@pytest.mark.skip("INDY-1911")
+def test_view_change_with_different_ic(looper, txnPoolNodeSet,
+                                       sdk_pool_handle,
+                                       sdk_wallet_client,
+                                       tconf, tdir, allPluginsPath):
+    """
+    1. panic_node (Delta) send InstanceChange for all nodes.
+    2. Restart nodes_to_restart (Beta, Gamma).
+    3. nodes_to_restart send InstanceChanges for all nodes.
+    4. Ensure elections done.
+    """
+    nodes_to_restart = txnPoolNodeSet[1:3]
+    panic_node = txnPoolNodeSet[-1]
+    view_no = txnPoolNodeSet[0].viewNo
+
+    panic_node.view_changer.on_master_degradation()
+    for n in nodes_to_restart:
+        _restart_node(looper, txnPoolNodeSet, n, tconf, tdir, allPluginsPath)
+    nodes_to_restart = txnPoolNodeSet[1:3]
+
+    for n in nodes_to_restart:
+        n.view_changer.on_master_degradation()
+
+    def check():
+        assert panic_node.view_change_in_progress
+
+    looper.run(eventually(check))
+    ensureElectionsDone(looper=looper, nodes=txnPoolNodeSet)
+    ensure_all_nodes_have_same_data(looper, nodes=txnPoolNodeSet)
+
+    for node in txnPoolNodeSet:
+        assert node.viewNo == view_no
```

Comment:
Since we are already relying in this test on a fact that view change starts on some nodes I think it's better to do something like this:
```suggestion
        assert node.viewNo > view_no
```
Another option is to just get rid of this check, since actually all that we want here is to check that nodes will eventually come to consensus after some panics and restarts, and this is already done by `ensureElectionsDone` and `ensure_all_nodes_have_same_data`.

Reply:
Done